examples/webpack.config.js

```diff
@@ -7,14 +7,15 @@ const path = require('path');
 const MonacoEditorSrc = path.join(__dirname, '..', 'src');
 
 module.exports = {
-  entry: {
```

Comment:
The advantage of using bundle splitting is that you get better performance. However, this is only an example and the simpler it is, the better. I also tried to make changes and the builds are almost instant even without bundle splitting.